packages/security/carbon.yml

```diff
@@ -0,0 +1,313 @@
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-combobutton
+      tags:
+        - input-control
+  filter-panel:
+      name: Filter panel
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/filter-panel.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-filterpanel
+      tags:
+        - form
+  icon-button-bar:
+      name: Icon button bar
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/icon-button-bar.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-iconbuttonbar
+      tags:
+        - input-control
+  non-entitled-section:
+      name: Non entitled section
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/non-entitled-section.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-nonentitledsection
+      tags:
+        - content-block
+  search-bar:
+      name: Search bar
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/search-bar.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-searchbar
+      tags:
+        - input-control
+        - form
+  status-indicator:
+      name: Status indicator
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/status-indicator.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-statusindicator
+      tags:
+        - system-feedback
+  tag-wall:
+      name: Tag wall
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/tag-wall.svg'
+      demoLinks:
+        - type: storybook
+          name: Storybook
+          action: link
+          url: https://carbon-for-ibm-products.netlify.app/?path=/docs/security-patterns-tagwall
+      tags:
+        - input-control
+        - data-display
+  tag-wall-filter:
+      name: Tag wall filter
+      status: stable
+      type: pattern
+      platform: web
+      framework: react
+      thumbnailPath: './thumbnails/tag-wall-filter.svg'
```

Comment:
missing thumb. do patterns not have thumbnails? if that's the case let's remove the "thumbnailPath" property